src/Console/Application.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Console;
+
+use Infection\Command;
+use Infection\Config\InfectionConfig;
+use Pimple\Container;
+use Symfony\Component\Console\Application as BaseApplication;
+use Symfony\Component\Console\Formatter\OutputFormatterStyle;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class Application extends BaseApplication
+{
+    const NAME = 'Infection - PHP Mutation Testing Framework';
```

Comment:
instead of having a constant here what about putting it in the constructor in `bin/infection`? It allows to pick a different name during the tests for example if you want to

Reply:
you could also leverage default values in the constructor instead:

```php
function __construct(
    Container $container,
    string $name = 'Infection - PHP Mutation Testing Framework',
    string $version = '@package_version@'
) {
    // ...
}
```

Reply:
yeah I thought about it but decided to extract this strings into constants. As for me it's static information which won't ever change.

@borNfreee what do you think?

give me a shout if you think that we should put these strings into constructor by default or move them into `bin/infection`

Reply:
> As for me it's static information which won't ever change.

I have no idea why we need to test/replace this string. It's up to you guys. Current implementation is ok IMO

Reply:
I like to do it like that because avoids a constant declaration and is aligned with a couple of other projects I check. But I really don't think it matters all to much it's just personal preferences so roll with whatever you prefer :)